pkg/apis/pipeline/v1/taskrun_types.go

```diff
@@ -179,6 +179,9 @@ const (
 	// TaskRunReasonResolvingTaskRef indicates that the TaskRun is waiting for
 	// its taskRef to be asynchronously resolved.
 	TaskRunReasonResolvingTaskRef = "ResolvingTaskRef"
+	// TaskRunReasonResolvingStepActionRef indicates that the TaskRun is waiting for
+	// its StepAction's Ref to be asynchronously resolved.
+	TaskRunReasonResolvingStepActionRef = "ResolvingRef"
```

Comment:
Just to be specific:

```suggestion
	TaskRunReasonResolvingStepActionRef = "ResolvingStepActionRef"
```